### PR TITLE
修复TDengineCommand通过无参构造函数预先初始化IStmt异常情况

### PR DESCRIPTION
**Pull request by d4ilys**
在使用ado.net时，会有先初始化Command然后在Connection Open执行的情况
~~~C#
//先初始化Command
var command = new TDengineCommand();
command.CommandText = "select * from `test`.`test_table01` ";

//然后开启链接执行Command
var connection =
    new TDengineConnection(
        "host=localhost;port=6030;username=root;password=taosdata;protocol=Native;db=test;");
connection.Open();

command.Connection = connection;

var dbDataReader = command.ExecuteReader();

Assert.True(dbDataReader.HasRows);
~~~

### CODE REVIEW

File: test/Data.Tests/TDenginePrepareCommandTests.cs
Comment (huskar-t) [inline test/Data.Tests/TDenginePrepareCommandTests.cs:17]: The database "test" does not exist, please create the database first

File: test/Data.Tests/TDenginePrepareCommandTests.cs
Comment (huskar-t) [inline test/Data.Tests/TDenginePrepareCommandTests.cs:34]: The database "test" does not exist, please create the database first

**Comment by huskar-t** [comment]
@d4ilys Thanks for the contribution, please see my comment above

**Comment by d4ilys** [comment]
@huskar-t  

In unit testing, I manually created the database locally

Please review TDengineCommand.cs

Or add a create database statement in unit testing after merging

**Comment by huskar-t** [comment]
> @huskar-t
> 
> In unit testing, I manually created the database locally
> 
> Please review TDengineCommand.cs
> 
> Or add a create database statement in unit testing after merging

@d4ilys Please create the database in the unit test. It can be merged after the unit test passes.

**Comment by 2881099** [comment]
请问，怎么创建数据库呢？

TDengineCommand 对象未设置 Connection 或者 Connection 未使用 Open 打开连接，都会报错 null object，例如：

var command = new TDengineCommand();

command.CommandText = "select * from `test`.`test_table01` ";

这个 PR 解决了该问题。

**Comment by huskar-t** [comment]
> 请问，怎么创建数据库呢？
> 
> TDengineCommand 对象未设置 Connection 或者 Connection 未使用 Open 打开连接，都会报错 null object，例如：
> 
> var command = new TDengineCommand();
> 
> command.CommandText = "select * from `test`.`test_table01` ";
> 
> 这个 PR 解决了该问题。

Create a database using a connection without db and then close the connection and delete the database after execution

**Review by huskar-t — APPROVED** [review]

**Pull request merged**
